walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,64 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.logging.Logger;
 
+/* FindMeetingQuery determines if people can have meetings based on each persons avaliable times */
 public final class FindMeetingQuery {
+
+  private static Logger log = Logger.getLogger(FindMeetingQuery.class.getName());
+
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    throw new UnsupportedOperationException("TODO: Implement this method.");
+
+    Collection<TimeRange> available = Arrays.asList(TimeRange.WHOLE_DAY);
+    Collection<String> attendees = new ArrayList(request.getAttendees());
+    
+    /* If meeting is longer than the whole day, return empty list */
```

Comment:
/* */ comments are used for class descriptions. Comments inside a class should use //. Applies to all comments in query()